.web-docs/components/builder/tart/README.md

```diff
@@ -50,6 +50,7 @@ For more advanced examples, please refer to the [`example/` directory](https://g
 - `cpu_count` (number) - Amount of virtual CPUs to use for the new VM. Overrides `tart create` default value when using `from_ipsw` and `from_iso` and VM settings when using `vm_base_name`.
 - `create_grace_time` (duration string | ex: "1h5m2s") — Time to wait after finishing the installation process. Can be used to work around the issue when Virtualization.Framework's installation process is still running in the background for some time after `tart create` had already finished.
 - `disk_size_gb` — Disk size in GB to use for the new VM. Overrides `tart create` default value when using `from_ipsw` and `from_iso` and VM settings when using `vm_base_name`.
+- `recovery_partition` (string) — Behavior with the respect to the macOS recovery partition. Set to `"delete"` to delete it (allows for disk resize, decreases the VM image size, but prevents the updates from working), `"keep"` to keep it as is (prevents the disk resize, increases the VM image size, but allows the updates to proceed) or `"relocate"` to move the partition to the end of disk (allows for disk resize, increases the VM image size, but allows the updates to proceed). Defaults to `""` which is currently equivalent to `"delete"`, but this may change in the future.
```

Comment:
```suggestion
- `recovery_partition` (string) — Behavior with the respect to the macOS recovery partition. Set to `"delete"` to delete it (allows for disk resize, decreases the VM image size, but prevents the macOS updates from working), `"keep"` to keep it as is (prevents the disk resize, increases the VM image size, but allows the macOS updates to proceed) or `"relocate"` to move the partition to the end of disk (allows for disk resize, increases the VM image size, but allows the macOS updates to proceed). Defaults to `""` which is currently equivalent to `"delete"`, but this may change in the future.
```